frontend/src/lib/Apis.ts

```diff
@@ -433,9 +433,9 @@ export class Apis {
     );
   }
 
-  public static async listContributors(): Promise<any> {
+  public static async listContributors(fUser: string): Promise<any> {
     const owner = localStorage.getItem('user') || ''
-    return this._fetchAndParse("/kfam/v1/bindings?namespace="+owner);
+    return this._fetchAndParse("/kfam/v1/bindings?namespace="+owner+(fUser?"&user="+fUser:""));
   }
```

Comment:
What is the use case ? On Contributors table, user is searching for contributors in the search box ? fUser is the search term ? we dont have user full name. can we just rename the variable to user here ?